ple/games/donkeykong/Game.py

```diff
@@ -0,0 +1,210 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import sys
+from pygame.locals import *
+from Board import Board
+from pygame import mixer
+from .. import base
+
+'''
+This class defines the logic of the game and how player input is taken etc
+We run one instance of this class at the start of the game, and this instance manages the game for us.
+'''
+
+
+class Game(base.Game):
+    def __init__(self):
+        '''
+        Set the height and width for the game element
+        FPS is set to 30 frames per second
+        '''
+        self.height = 520
+        self.width = 1200
+
+        actions = {
+                "left": K_a,
+                "right": K_d,
+                "jump": K_SPACE,
+                "climb": K_w
+        }
+        base.Game.__init__(self, self.width, self.height, actions=actions)
+
+        self.FPS = 30
+        self.clock = pygame.time.Clock()
+        self.displayScreen = pygame.display.set_mode((self.width, self.height))
```

Comment:
Clock and the display screen should not be set by the game itself. Also when you are calling the screen within your game please use `self.screen` instead of `self.displayScreen` or create an alias internally